pages/support.tsx

```diff
@@ -0,0 +1,241 @@
+import React from 'react';
+import Head from 'next/head';
+import Footer from 'src/views/home/Footer';
+import Image from 'next/image';
+
+type ChildProp = {
+  children: React.ReactElement | Array<React.ReactElement | string> | string;
+  className?: string;
+}
+
+const Heading = function ({ children, className }: ChildProp): JSX.Element {
+  return (<h2 className={'font-bold font-sans text-3xl lg:text-6xl capitalize text-transparent bg-clip-text bg-gradient-to-r from-white to-[#808080]' + ' ' + className}>
```

Comment:
Use `classNames` utility to combine class names, like `className={classNames(a, b, c, ...)}`

Reply:
✅